vllm/config.py

```diff
@@ -237,7 +237,8 @@ def _verify_quantization(self) -> None:
                     f"{self.quantization} quantization is currently not "
                     f"supported in ROCm.")
             if (self.quantization
-                    not in ("fp8", "marlin", "gptq_marlin_24", "gptq_marlin")):
+                    not in ("fp8", "marlin", "gptq_marlin_24", "gptq_marlin",
+                            "compressed-tesnors")):
```

Comment:
```suggestion
                            "compressed-tensors")):
```
also is it not an underscore?

Reply:
Yeah it seems underscore https://github.com/vllm-project/vllm/blob/724d7e901c1e88f8a9f3d182ec876ffc589a3bb3/vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py#L42C17-L42C35